prime-router/settings/staging/0169-epic-staging-sender-receiver.yml

```diff
@@ -0,0 +1,112 @@
+# To submit this to localhost, run:
+# ./prime multiple-settings set --env staging --input ./settings/staging/0169-epic-staging-sender-receiver.yml
+#
+# Description:
+#   These following setting are for Epic Systems Integration Testing
+#   Order sender (epic.etor-nbs-orders)
+#   Result Receiver (epic.etor-nbs-results)
+#
+---
+- name: "epic"
+  description: "Epic Systems Integration Testing"
+  jurisdiction: "STATE"
+  stateCode: LA
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: null
+  senders:
+    - !<full-elr>
+      name: "etor-nbs-orders"
+      organizationName: "epic"
+      format: "HL7"
+      customerStatus: "active"
+      schemaName: ""
+      processingType: "async"
+      allowDuplicates: true
+      senderType: null
+      primarySubmissionMethod: null
+      topic: "etor-ti"
+      version: null
+      createdBy: null
+      createdAt: null
+  receivers:
+    - name: "etor-nbs-results"
+      organizationName: "epic"
+      topic: "etor-ti"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "metadata/hl7_mapping/ORU_R01/ORU_R01-base"
```

Comment:
The schema here will be `metadata/hl7_mapping/receivers/Flexion/TIResults` once PR #13308 gets merged. BTW, that PR is ready for review so you can take a look and approve if everything looks good to you

Reply:
FYI, I removed `TIResults` schema for now, so you can leave the schema as is. No need to update it